starters/tailwind/src/Disclosure.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useContext } from "react";
+import {
+  UNSTABLE_Disclosure as AriaDisclosure,
+  UNSTABLE_DisclosureGroup as AriaDisclosureGroup,
+  DisclosureProps as AriaDisclosureProps,
+  DisclosureGroupProps as AriaDisclosureGroupProps,
+  UNSTABLE_DisclosurePanel as AriaDisclosurePanel,
+  DisclosurePanelProps as AriaDisclosurePanelProps,
+  composeRenderProps,
+  Heading,
+  Button,
+  DisclosureStateContext,
+} from "react-aria-components";
+import { tv } from "tailwind-variants";
+import { ChevronRight } from "lucide-react";
+import { composeTailwindRenderProps, focusRing } from "./utils";
+import { DisclosureGroupStateContext } from "react-aria-components";
+
+const disclosure = tv({
+  base: "w-64 border rounded-lg bg-gray-100 text-gray-600 border-gray-200 hover:border-gray-300 dark:bg-zinc-700 dark:text-zinc-300 dark:border-zinc-600 dark:hover:border-zinc-500 outline-none",
+});
+
+const disclosureInGroup = tv({
+  extend: disclosure,
+  base: "border-0 border-b last:border-b-0 rounded-b-none last:rounded-b-lg",
+});
+
+const disclosureButton = tv({
+  extend: focusRing,
+  base: "rounded flex gap-2 items-center w-full text-start p-2 cursor-default",
+  variants: {
+    isDisabled: {
+      true: 'text-gray-300 dark:text-zinc-600 forced-colors:text-[GrayText]'
+    },
+  },
+});
+
+const chevron = tv({
+  base: "w-5 h-5 text-gray-500 dark:text-gray-400 transition-transform duration-200 ease-in-out",
+  variants: {
+    isExpanded: {
+      true: "transform rotate-90",
```

Comment:
Not sure why [`rotate-90`](https://tailwindcss.com/docs/rotate) isn't working here? `rotate-180` works, so not a state issue.

Reply:
it looks like it's rotating 90 degrees to me. are you seeing something different?

Reply:
Same as below, I'm seeing it now